parallel-letter-frequency/example.rs

```diff
@@ -41,9 +40,9 @@ pub fn frequency(texts: &[&str], num_workers: usize) -> HashMap<char, usize> {
 fn count(lines: Vec<String>) -> HashMap<char, usize> {
     let mut results: HashMap<char, usize> = HashMap::new();
     for line in lines.iter() {
-        for c in line.as_slice().chars() {
+        for c in line.chars() {
             if c.is_alphabetic() {
-                match results.entry(c.to_lowercase()) {
+                match results.entry(c.to_ascii_lowercase()) {
```

Comment:
I believe the [ASCII lowercase](http://doc.rust-lang.org/std/ascii/trait.AsciiExt.html#tymethod.to_ascii_lowercase) method does a check to see if the char is actually ASCII. Though if we included a test with both a upper and lower case Unicode character (example: "Θ θ"), I believe it would count these as separate characters...

I will update using the [Unicode version](http://doc.rust-lang.org/std/primitive.char.html#method.to_lowercase) which should, if I read the docs correctly, count Unicode upper and lower ("Θ θ") as the same character.